src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs

```diff
@@ -1032,7 +1038,7 @@ private static int GetPortV4(string responseString)
 
             int index = parsedList.Length - 1;
             // skip the last non-number token (e.g. terminating '.')
-            if (!char.IsNumber(parsedList[index], 0))
+            if (!char.IsNumber(parsedList[index][0]))
```

Comment:
This is not the same thing.  The former will potentially process multiple characters starting at position 0 whereas the latter will only process the single character supplied. For example, this:
```C#
char.IsNumber("\U000102E7", 0)
```
will return true, whereas:
```C#
char.IsNumber("\U000102E7"[0])
```
will return false.  I don't know whether the intent of this code was only to support ASCII digits, in which case your change would be fine, but our test coverage of this code isn't great, and so we shouldn't change it.  There's also no need to change it.  Please revert.

Reply:
Oops, it was not intentional, typo from revert.